build.gradle

```diff
@@ -1,21 +1,26 @@
 buildscript {
-    ext.kotlinVersion = '1.2.41'
-    ext.supportLibVersion = '27.1.1'
-    ext.lifecycleVersion = '1.1.1'
+    ext.kotlinVersion = '1.2.51'
+    ext.androidxVersion = '1.0.0'
```

Comment:
While this works, they are being updated independently (there is already, for example, androix.appcompat 1.0.2, which fixes important issues with PrecomputedTextCompat, while RecyclerView is still at 1.0.0)

Reply:
Good point and I wanted to break this outer later. I did not realize that an independent bugfix release already happened. Let me update that.